dist/README.md

```diff
@@ -182,41 +182,14 @@ For further instructions on these variables as well as configuring
 authentication, consult [our documentation for
 self-hosting](https://support.getgrist.com/self-managed/).
 
-# updating Grist
+# Upgrading Grist
 
-To update to the latest Grist version at any time, first stop Grist.
-
-If using systemd:
-
-```sh
-sudo systemctl disable --now grist
-```
-
-Otherwise:
-
-```sh
-docker compose down
-```
-
-Then pull the latest changes:
-
-```sh
-docker compose pull
-```
-
-Once that completes successfully, restart Grist.
-
-If using systemd:
-
-```sh
-sudo systemctl enable --now grist
-```
-
-Otherwise:
+To update to the latest Grist version, we recommend using systemd as
+described above, and enabling the supplied systemd timer:
 
 ```sh
-docker compose up
+sudo systemctl enable --now grist-update.timer
```

Comment:
nice to be automating this, but also feels useful to tell someone how to upgrade directly or to know which version they have now.

Reply:
I would like to address this request at a later time. For now users can check the running version in the admin, although that only gives the tagged version, not the Git commit that was used to build the Docker image that is currently running.